bin/celeste_mcmc.jl

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env julia
+
+# ./celeste.jl infer-box 200 200.5 38.1 38.35
+using Distributions
+import Celeste: Model, Infer, RunCamcolField, load_images, init_source
+import Celeste.Model: CatalogEntry, TiledImage, ElboArgs,
+                      get_active_pixels
+
+# Mamba for MCMC tools
+import Mamba
```

Comment:
Or alternatively we could just include the log probability function, and unit tests for it, in this PR. Either way. But I suppose if we're going to have the MCMC code in this PR, it should be committed with tests.